### PR TITLE
add save / load authenticated session js sdk example.  

### PR DESCRIPTION
**Pull request by lozzle**
ran script and saw successful reload.

i think there's some js-ism way to open and close the browser session but i'm not totally sure.  please educate me if there's a better way to do this!

### CODE REVIEW

File: javascript-sdk/examples/save-and-load-authenticated-session/save-and-load-authenticated-session.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+const { wrap, configure } = require('agentql');
+const { chromium } = require('playwright');
+
+(async () => {
+  // Set the AgentQL API key via the `configure` method.
+  require('dotenv').config({ path: '.env.local' })
+  configure({ apiKey: process.env.API_KEY });
```
Comment (frankfeng98) [inline javascript-sdk/examples/save-and-load-authenticated-session/save-and-load-authenticated-session.js:7]: @rachelnabors and I are thinking about removing `configure` line and using `dotenv` line from examples. JS SDK currently supports directly read in AGENTQL_API_KEY as a backup option when retrieving API key. We will ask users to set API key as an environment variable in installation guide.

File: javascript-sdk/examples/save-and-load-authenticated-session/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Example script: save and load authenticated state with AgentQL
+
+This example demonstrates how to save and load a authenticated state (i.e. signed-in state) using AgentQL.
+
+## Run the script
+
+- [Install AgentQL SDK](https://docs.agentql.com/installation/sdk-installation)
+- Save this python file locally as **save-and-load-authenticated-state.js**
```
Comment (frankfeng98) [inline javascript-sdk/examples/save-and-load-authenticated-session/README.md:8]: Javascript file

File: javascript-sdk/examples/save-and-load-authenticated-session/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+- Run the following command from the project's folder:
+
+```bash
+node save_and_load_authenticated_state.py
```
Comment (frankfeng98) [inline javascript-sdk/examples/save-and-load-authenticated-session/README.md:12]: save-and-load-authenticated-state.js

**Review by frankfeng98 — CHANGES_REQUESTED** [review]
The format issue could be resolved by CD into `javascript-sdk` folder, run `npm install` and then `npm run format`

**Comment by lozzle** [comment]
sorry sorry, swapping between sapling and git confused me.  i think it's a bit better this time

**Review by frankfeng98 — APPROVED** [review]

**Comment by lozzle** [comment]
sorry, also lost the changes where i ported in comments from the python version of this.  re-added that!

**Pull request merged**